### ISSUE TITLE
[Question] Can sites outside of my container see the cookies of the website inside the container?

### ISSUE THREAD

**Issue by tomervazana** [arkenfox/user.js#1882]
I find it really annoying to authenticate my Google account from my phone every time I want to log in, so I decided to allow cookies from Google. However, Google has its own dedicated container.

1. Can sites outside of the Google container now see my browsing activity on Google?
2. Can Google see my browsing activity outside of its own container?
3. Is there any way I can use Google sign-in (for example to sign in to ChatGPT) sync while maintaining the container?
4. Can I achieve the same result on LibreWolf with settings similar to Arkenfox? (I really like Arkenfox but find LibreWolf easier to maintain, and as far as I understand, they aim to achieve the same results without breaking sites. It really fits my needs.)

![Screenshot 2024-08-17 073214](https://github.com/user-attachments/assets/8e1b97f6-1ac4-4850-8a91-8ceb2a773442) ![Screenshot 2024-08-17 073305](https://github.com/user-attachments/assets/2ff70d1a-d295-4079-bcf4-748cc73f79e1)
![Screenshot 2024-08-17 072252](https://github.com/user-attachments/assets/71634b67-6309-4d98-8dae-6235fbb76644)

Thank you so much for this amazing project, for your help, and for making the internet safer place!

**Comment by rusty-snake** [comment]
1./2. Cookies are isolated. If you open site A "normal" tab and in a containerd tab (let's call it A*), then has A no access to the cookies set by A* and vice versa.
3. To me it's unclear what you mean with "while maintaining the container". To use Google sign-in, open ChatGPT in you Google container.

**Comment by tomervazana** [comment]
> 1./2. Cookies are isolated. If you open site A "normal" tab and in a containerd tab (let's call it A*), then has A no access to the cookies set by A* and vice versa. 3. To me it's unclear what you mean with "while maintaining the container". To use Google sign-in, open ChatGPT in you Google container.

What I meant in 3 is: Is it mandatory to log into ChatGPT from the Google container tab if I want to use Google sign-in? 
From your answer, I understand that it is.

Thank you!

**Comment by sojusnik** [comment]
>     1. Can sites outside of the Google container now see my browsing activity on Google?
>     2. Can Google see my browsing activity outside of its own container?
Yes, if you set Firefox to clear cookies/data after closing, due to [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1767271), see also [this](https://reddit.com/r/firefox/comments/1k464n9/eli5_cookieclearing_exceptions_affecting_cookie/) and [this](https://github.com/arkenfox/user.js/issues/1885) discussion.
